resources/views/review.blade.php

```diff
@@ -1,3 +1,58 @@
-<x-layout>
-    <h1>Review</h1>
+@php
+    $formattedDate = formatDate($review->created_at);
+    $backLabel = request('from') === 'movie' ? 'Back to movie' : (request('from') === 'profile' ? 'Back to profile' : 'Back');
+    $backHref = url()->previous();
+@endphp
+
+<x-layout class="pt-1 sm:pt-10">
+    <div class="flex items-start justify-between gap-6">
+        <x-section-header.back-link
+            :title="$review->movie->title"
+            extraLabel="Review"
+            href="{{ $backHref }}"
+            backLabel="{{ $backLabel }}"
+        />
+        <x-button
+            variant="icon"
+            srLabel="Open review menu"
+            class="hidden sm:block"
+        >
+            <x-lucide-ellipsis-vertical
+                class="size-6 cursor-pointer text-slate-50"
```

Comment:
```suggestion
                class="size-6 text-slate-50"
```

handled by the button